src/pt/wolftoon/src/eu/kanade/tachiyomi/extension/pt/wolftoon/Wolftoon.kt

```diff
@@ -0,0 +1,248 @@
+            .addQueryParameter("title_id", "eq.$titleId")
+            .addQueryParameter("order", "chapter_number.desc")
+            .fragment(titleId)
+            .build()
+
+        return GET(url, apiHeaders)
+    }
+
+    override fun chapterListParse(response: Response): List<SChapter> =
+        response.parseAs<List<ChapterDto>>().map { it.toSChapter() }
+
+    // ================================ Pages =======================================
+
+    override fun fetchPageList(chapter: SChapter): Observable<List<Page>> {
+        return Observable.fromCallable {
+            val chapterUrl = getChapterUrl(chapter).toHttpUrl()
+            val chapterId = chapterUrl.fragment
+            val titleId = chapterUrl.pathSegments[1]
+            val url = "$supabaseUrl/rest/v1/chapters".toHttpUrl().newBuilder()
+                .addQueryParameter("select", "id,title_id,images")
+                .addQueryParameter("title_id", "eq.$titleId")
+                .addQueryParameter("order", "chapter_number.desc")
+                .fragment(chapterId)
+                .build()
+            pageListParse(client.newCall(GET(url, apiHeaders)).execute())
+        }
+    }
+
+    override fun pageListParse(response: Response): List<Page> {
+        val chapterId = response.request.url.fragment
+        return response.parseAs<List<PageDto>>()
+            .firstOrNull { it.id == chapterId }
+            ?.toPageList()
+            ?: emptyList()
+    }
+
+    override fun imageUrlParse(response: Response): String = ""
+
+    // ================================ Filters =======================================
+
+    private var genreList: Array<String> = emptyArray()
+    private fun getGenreList() {
+        if (genreList.isNotEmpty()) {
+            return
+        }
+
+        val script = client.newCall(GET(scriptUrl, headers)).execute().body.string()
+        genreList = arrayOf("Todos") + (
+            GENRE_REGEX.find(script)?.groupValues?.last()
+                ?.parseAs<Array<String>>()
+                ?: emptyArray()
+            )
+    }
+
+    private val scope = CoroutineScope(Dispatchers.IO)
+
+    override fun getFilterList(): FilterList {
+        scope.launch { getGenreList() }
+        val filters = mutableListOf<Filter<*>>(
+            StatusFilter("Status", statusList),
+            TypeFilter("Tipo", typeList),
+            OrderBy("Ordenar", orderByList),
+        )
+
+        when {
+            genreList.isNotEmpty() -> filters += GenreFilter("Gêneros", genreList)
+            else -> filters += listOf(
+                Filter.Separator(),
+                Filter.Header("Aperte 'Redefinir' para tentar mostrar os gêneros"),
+            )
+        }
+
+        return FilterList(filters)
+    }
+
+    companion object {
+        private val GENRE_REGEX = """Che=(\[[^]]+])""".toRegex()
+        private val API_KEY_REGEX = """bse="([^"]+)""".toRegex()
+        val DATE_FORMAT = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss")
```

Comment:
Add timestamp if known, otherwise it will default to the user's timezone.

Reply:
My bad- I meant to say you should add the *timezone* if known. Adding `Locale.ROOT` makes it more explicit, but doesn't change the timezone itself.

Normally you would add timezone like this:
```kt
165-    private val cursorDateFormat = SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.US).apply {
166:        timeZone = TimeZone.getTimeZone("UTC")
167-    }
```